### ISSUE TITLE
URL error

### ISSUE THREAD

**Issue by gbywt** [tschellenbach/Django-facebook#585]
File "C:/Users/User/Downloads/myprojects\myprojects\urls.py", line 27, in <module>
    url(r'^facebook/', include('django_facebook.urls')),
  File "C:\Users\User\.virtualenvs\myprojects\lib\site-packages\django\conf\urls\__init__.py", line 50, in include
    urlconf_module = import_module(urlconf_module)
  File "C:\Python27\Lib\importlib\__init__.py", line 37, in import_module
    __import__(name)
  File "C:\Users\User\.virtualenvs\myprojects\lib\site-packages\django_facebook\urls.py", line 4, in <module>
    from django.conf.urls.defaults import patterns, url
ImportError: No module named defaults

Is that setup is not updated ?

**Comment by vvolodin** [comment]
Yes, it's been deprecated for quite some time now. I think this project is dead now.
